api/controllers/game/join-and-start-new-game.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { newGameId, pNum } = req.body;
+    const game = await Game.findOne({ id: newGameId }).populate('players');
+    Game.subscribe(req, [game.id]);
+
+    req.session.game = game.id;
+    req.session.pNum = pNum;
+
+    return res.ok();
```

Comment:
I think it makes sense to emit the game's lastevent here so any client who missed it can get the last event an reconstruct the current game state

Reply:
Update the `join-rematch` API, not quite sure if it fully captures what you mean in the comment